config/routes.rb

```diff
@@ -18,16 +18,18 @@
 
   # LetsEncrypt
   get '/.well-known/acme-challenge/:id' => 'pages#letsencrypt'
-
   get '/veterans/thanks', to: 'veterans#thanks'
-  get '/veterans/map', to: 'veterans#map'
```

Comment:
I'm OK with changing the URL but I think we should maintain the old one (or at least 301 to the new URL) in case anyone has linked to it.